common/src/main/java/io/github/consistencyplus/consistency_plus/blocks/BlockColors.java

```diff
@@ -0,0 +1,35 @@
+package io.github.consistencyplus.consistency_plus.blocks;
+
+import net.minecraft.block.MapColor;
+
+import java.util.Locale;
+
+public enum BlockColors {
+    RED, ORANGE, YELLOW, LIME, GREEN, BLUE, CYAN, LIGHT_BLUE, PURPLE, MAGENTA, PINK, WHITE, LIGHT_GRAY, GRAY, BLACK, BROWN;
```

Comment:
Surely there's already a vanilla enum for this? `Formatting`, right?

Reply:
Not for these colors, no, these are the dye colors.

Reply:
Take that back, there is a dye color enum.

Reply:
Only problem with that would be the fact that it doesnt support terracotta colors, which can probably be bypassed.

(As a side note, it also showed me what Jay meant by including values in Enums.)